httpx/config.py

```diff
@@ -39,9 +39,16 @@ class SSLConfig:
     SSL Configuration.
     """
 
-    def __init__(self, *, cert: CertTypes = None, verify: VerifyTypes = True):
+    def __init__(
```

Comment:
Let's revert usages of trust_env for SSLConfig? The default behavior in the future should be use system trust stores unless we're specifically told which cert bundle to use.

Reply:
You're right new approach for environment variables would be nice.
I reverted the last commit.